WalletWasabi/Blockchain/TransactionOutputs/SmartCoin.cs

```diff
@@ -132,6 +133,10 @@ public bool IsExcludedFromCoinJoin
 		set => RaiseAndSetIfChanged(ref _isExcludedFromCoinJoin, value);
 	}
 
+	/// <returns>False if external, or the tx inputs are all external.</returns>
+	/// <remarks>Context: https://github.com/zkSNACKs/WalletWasabi/issues/10567</remarks>
+	public bool IsSufficientlyDistancedFromExternalKeys { get; set; } = true;
```

Comment:
Most of the time when we receive a new coin, it comes from an external source.

Shouldn't we use `false` as default value?

Reply:
That's a valid argument. I believe there's a more important argument to be made for `true` from the point of view of security though: If we make a mistake, let's make a mistake towards not draining the user's wallet.